fix/empty-wheelbarrows.lua

```diff
@@ -0,0 +1,41 @@
+--checks all wheelbarrows on map for rocks stuck in them. If a wheelbarrow isn't in use for a job (hauling) then there should be no rocks in them
+--rocks will occasionally get stuck in wheelbarrows, and accumulate if the wheelbarrow gets used.
+--this script empties all wheelbarrows which have rocks stuck in them.
+
+local args = {...}
+
+local quiet = false
+local dryrun = false
+for i,arg in ipairs(args) do
+    if (arg == "--quiet") then
+        quiet = true
+    end
+    if (arg == "--dryrun") then
+        dryrun = true
+    end
+end
```

Comment:
I would recommend using `utils.processArgsGetopt()` for this. Among other benefits, it supports "short" options like `-q` (if specified) and produces errors for unrecognized options.

Reply:
So something like this?

```
local cmds = argparse.processArgsGetopt(args, {
    {'q', 'quiet', handler=function() quiet = true end},
    {'n', 'dryrun', handler=function() dryrun = true end},
})
```